src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryText.cs

```diff
@@ -1187,6 +1187,46 @@ private static DateTime ConvertToUTC(DateTime dateTime, TimeZoneInfo fromTimeZon
             return resultDateTime.UtcDateTime;
         }
 
+        public static FormulaValue Unichar(IRContext irContext, NumberValue[] args)
+        {
+            double number = args[0].Value;
+
+            // Unicode upper and lower bounds
+            const double lowerBound = 1; // Excel returns error for Unichar(0)
+            const double upperBound = 1114111; // 0x10FFFF

```

Comment:
Excel returns #NA for 1114111 and 1114110 - UNICHAR(1114109) returns a valid value. C# and JS do support those values, so I think we're ok in supporting them in Power Fx.

Reply:
FWIW, I would use hex notation in the code itself instead of using the decimal form and a comment